nss-day-cohort-48/the-worst-site-ever-fandango
# Removing extra pages

Hello friends,

## Steps to Test

1. Pull this branch gqg-headers-1
2. Open in Chrome, serve
3. Open in VS Code
4. Verify that chaos-index.html is removed and CSS files all reside in the Styles directory. 

Thanks!
